sky/tools/create_macos_framework.py

```diff
@@ -223,6 +229,34 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
     final_dst_path = os.path.join(dst, 'FlutterMacOS.framework.zip')
     shutil.move(final_src_path, final_dst_path)
 
+    zip_xcframework_archive(dst)
+
+
+def zip_xcframework_archive(dst):
+  filepath_with_entitlements = ''
+  filepath_without_entitlements = (
+      'FlutterMacOS.xcframework/macos-arm64_x84_64/'
+      'FlutterMacOS.framework.zip/Versions/A/FlutterMacOS'
```

Comment:
Why is there a zip inside the xcframework?

Reply:
Oops, that was a copy mistake. Good catch